coredns/README.md

```diff
@@ -2,20 +2,21 @@
 
 ## Overview
 
-Get metrics from CoreDNS in real time to visualize and monitor DNS failures and cache hits/misses.
+Get metrics from CoreDNS in real time to visualize and monitor DNS failures and cache hits or misses.
 
 ## Setup
 
-### Installation
+### Prerequisites
 
-The CoreDNS check is included in the [Datadog Agent][1] package, so you don't need to install anything else on your servers.
+This OpenMetrics-based integration has a latest version (V2) and a legacy version (V1). To get all the most up-to-date features, Datadog recommends upgrading to the latest version. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][21].
```

Comment:
```suggestion
Starting with version 1.11.0, this OpenMetrics-based integration has a latest version (V2) and a legacy version (V1). To get all the most up-to-date features, Datadog recommends upgrading to the latest version. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][21].
```

I do worry somewhat about the confusion that could arise from overloading the "version" word to mean both the integration's version (1.11.0) vs the OpenMetrics legacy / latest thing.

Reply:
How about this?